server/src/core/mongo.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require("mongoose");
 require('dotenv').config()
-const isMongoDbUrl = Boolean(process.env.IS_MONGODB_CLOUD_URL);
+const isMongoDbUrl = JSON.parse(process.env.IS_MONGODB_CLOUD_URL);
```

Comment:
if `IS_MONGODB_CLOUD_URL` is `undefined` then we will face the same problem. Let's try `JSON.parse(a ? a : 'false')` algo for now. 😎
Thanks 👍

Reply:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/497812/155869341-c8aeb9d5-2574-4c28-8e8e-d7aaf94333c3.png">